### PR TITLE
Deprecate `Controller#content` alias

### PR DESCRIPTION
**Pull request by locks**
- [x] Removes `ControllerContentModelAliasDeprecation`-related code.
- [x] Deprecates the `Controller#content` alias.
  - [x] Use `deprecatingAlias`.
  - [x] Write [deprecation guide](https://github.com/emberjs/website/pull/2969).

### CODE REVIEW

File: packages/ember-runtime/lib/mixins/controller.js [hunk 1]
```diff
@@ -44,6 +44,6 @@ export default Mixin.create(ActionHandler, ControllerContentModelAliasDeprecatio
   /**
     @private
   */
-  content: alias('model')
+  content: deprecatingAlias('model')
```
Comment (rwjblue) [inline packages/ember-runtime/lib/mixins/controller.js]: This will need an options hash with id, until, description, etc

**Review by rwjblue — CHANGES_REQUESTED** [review]

**Comment by rwjblue** [comment]
Can you also remove the entry in features.json?

**Comment by locks** [comment]
@rwjblue please re-review and 👀 the commit message for correctness 🙇

File: packages/ember-runtime/lib/mixins/controller.js [hunk 2]
```diff
@@ -44,6 +44,5 @@ export default Mixin.create(ActionHandler, ControllerContentModelAliasDeprecatio
   /**
     @private
   */
-  content: alias('model')
-
+  content: deprecatingAlias('model', { id: 'ember-runtime.controller-content', until: '2.16.0' })
```
Comment (rwjblue) [inline packages/ember-runtime/lib/mixins/controller.js]: We also need to add a deprecation guide entry to the website and add a `url` property with the link.

**Review by rwjblue — COMMENTED** [review]
Looks good. Happy to land when green, but we still need to follow up with the deprecation guide stuff for the newly added deprecation.

File: packages/ember-runtime/lib/mixins/controller.js (hunk 2 above)
Comment (locks) [inline packages/ember-runtime/lib/mixins/controller.js]: deprecation guide PRed: https://github.com/emberjs/website/pull/2969

**Review by rwjblue — APPROVED** [review]

**Pull request merged**